spec/factories/categories.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :category do
```

Comment:
add general attributes and relations

Reply:
Apu, I have added them in the branch, `4-write-tests-for-models`.